test/onnx/test_pytorch_onnx_no_runtime.py

```diff
@@ -763,5 +764,24 @@ def check_proto():
         self.assertRaises(RuntimeError, check_proto)
 
 
+    def test_is_fp_for_C_TypeList(self):
+        class M(torch.nn.Module):
```

Comment:
Helpful to comment inline how this model ties to the nodes being tested (which line does what)

Reply:
No tests do that :)
Comments are great for explaining `why` of a change, not `how` (that is what the code does already)
Keeping code and comment synced is a mess, so details of `how` are left to the code only.